examples/csi-app.yaml

```diff
@@ -11,10 +11,22 @@ spec:
       volumeMounts:
         - mountPath: "/data"
           name: my-csi-volume
-      command: [ "sleep", "1000000" ]
+      command:
+        - sh
+        - -c
+        - |
+          while true
+          do
+            ls -la /data
+            touch /data/foo
```

Comment:
this is (possibly short term) proof around whether the volume is read only or not ... toggling the `readOnly: true` setting below would be a way for users to play around with this element of the system.